tests/e2e/pkg/common.go

```
@@ -47,7 +47,10 @@ func GetVerrazzanoPassword() string {
 }
 
 func GetVerrazzanoPasswordInCluster(kubeconfigPath string) string {
-	secret, _ := GetSecretInCluster("verrazzano-system", "verrazzano", kubeconfigPath)
+	secret, err := GetSecretInCluster("verrazzano-system", "verrazzano", kubeconfigPath)
+	if err != nil {
+		return ""
```

Should this be returning an error in this case.  It looks like the error will be lost.

---

or at least log the error